### PR TITLE
Add support for Ceph NFS Gateway

### PR DESCRIPTION
**Pull request by dang**
Ceph has the ability to export it's filesystem via NFS using Ganesha.
Add a ceph-nfs role that will start Ganesha and export the Ceph
filesystems.

Note that, although support is going in to export RGW via NFS, this is
not working yet.

Signed-off-by: Daniel Gryniewicz <dang@redhat.com>

### CODE REVIEW

File: roles/ceph-common/templates/ganesha.conf.j2 [hunk 1]
```diff
@@ -0,0 +1,20 @@
+EXPORT
```
Comment (leseb) [inline roles/ceph-common/templates/ganesha.conf.j2]: Can you add the `ansible_managed` flag? just like in `ceph.conf.j2`.

File: roles/ceph-nfs/LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!) The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [2014] [Sébastien Han]
```
Comment (leseb) [inline roles/ceph-nfs/LICENSE]: I'll leave the credits to you.

File: roles/ceph-nfs/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+Have a look at: `defaults/main.yml`.
+
+## Mandatory variables
+
+None.
+
+# Dependencies
+
+The role `leseb.ceph-common` must be installed.
```
Comment (leseb) [inline roles/ceph-nfs/README.md]: s/`leseb.ceph-common`/`ceph.ceph-common`/

File: roles/ceph-nfs/README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+**THIS REPOSITORY DOES NOT ACCEPT PULL REQUESTS**
+**PULL REQUESTS MUST GO THROUGH [CEPH-ANSIBLE](https://github.com/ceph/ceph-ansible)**
+
+# License
+
+Apache
+
+# Author Information
+
+This role was created by [Sébastien Han](http://sebastien-han.fr/).
```
Comment (leseb) [inline roles/ceph-nfs/README.md]: I leave the credits to you.

File: roles/ceph-nfs/tasks/docker/create_configs.yml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+---
+- name: create ganesha conf directory
+  file:
+      path: /etc/ganesha
```
Comment (leseb) [inline roles/ceph-nfs/tasks/docker/create_configs.yml]: please use 2 spaces indent.

File: test.yml [hunk 6]
```diff
@@ -7,4 +7,5 @@
     - ceph-osd
     - ceph-mds
     - ceph-rgw
+    - ceph-nfs
```
Comment (leseb) [inline test.yml]: This will be tested by the CI, I don't think we are ready to test this.

**Comment by dang** [comment]
Ping

**Pull request merged**